README.md

```diff
@@ -7,13 +7,13 @@ drone-wall
 
 ### Deployment
 
-    $ docker pull scottwferg/drone-wall
-    $ docker run -p 3000:3000 -e API_SCHEME=$API_SCHEME -e API_DOMAIN=$API_DOMAIN \
-        -e API_TOKEN=$API_TOKEN -e API_PORT=$API_PORT scottwferg/drone-wall
+    $ docker pull scottwferg/drone-wall:2.1
+    $ docker run -p 3000:8080 -e API_SCHEME=$API_SCHEME -e API_DOMAIN=$API_DOMAIN \
+        -e API_TOKEN=$API_TOKEN -e API_PORT=$API_PORT -e WALL_PORT=8080 scottwferg/drone-wall:2.1
 
 Drone wall exposes port `3000`. You can map this to whatever you like.
```

Comment:
I agree but I don't want to change the way people is used to work with the v2 branch so here I was trying just to demonstrate how could it be used